src/Console/Commands/MigrateCommand.php

```diff
@@ -19,23 +18,21 @@ final class MigrateCommand
     public function __construct(
         private readonly Console $console,
         private readonly MigrationManager $migrationManager,
-        private readonly AppConfig $config,
     ) {
     }
 
     #[ConsoleCommand(
         name: 'migrate:up',
+        aliases: ['haha'],
         description: 'Run all new migrations',
+        isDangerous: true,
+        help: [
```

Comment:
Why does help text need to be an array? Wouldn't a string suffice?

Reply:
I'd go with string | string[]. If user wants to create a longer description, or add some white space, this allows you for an easier control while still being convenient

Reply:
Wouldn't a heredoc be sufficient in that case? I'm just trying to imagine when an array would really be helpful, and I can't really come up with a good example.